fapi-client/src/main/scala/com/gu/facia/api/contentapi/ContentApi.scala

```diff
@@ -0,0 +1,89 @@
+package com.gu.facia.api.contentapi
+
+import java.net.URI
+
+import com.gu.contentapi.client.GuardianContentClient
+import com.gu.contentapi.client.model._
+import com.gu.facia.api.{CapiError, Response}
+
+import scala.concurrent.ExecutionContext
+
+
+object ContentApi {
+  type AdjustSearchQuery = SearchQuery => SearchQuery
+  type AdjustItemQuery = ItemQuery => ItemQuery
+
+  def buildHydrateQuery(client: GuardianContentClient, ids: List[String]): SearchQuery = {
```

Comment:
That's an excellent point. No, we're likely to run into problems with that at some point.